bin/test-documentation

```diff
@@ -17,6 +17,9 @@ fi
 cd docs
 runSbt markdownValidateDocs test markdownEvaluateSbtFiles $EXTRA_TASK
 
+runSbt +headerCheck \
+       +test:headerCheck \
+       multi-jvm:headerCheck
```

Comment:
We have `multi-jvm` in `docs/`?

Reply:
We don't. It was part of #1984. Fixed there and rebased here. :-)